common/djangoapps/util/course.py

```diff
@@ -57,3 +57,14 @@ def get_link_for_about_page(course):
         )
 
     return course_about_url
+
+
+def has_certificates_enabled(course):
```

Comment:
`has_certificates_enabled` is needed b/c in cms.urls.py we are constructing urls based on `settings.FEATURES.get('CERTIFICATES_HTML_VIEW', False)` feature flag. if that is `False` L#63 will fail  ` cms/templates/checklists.html`.